app.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const { getCategories } = require("./controller/seed.controller.js");
+const app = express();
+app.use(express.json());
```

Comment:
app.use(express.json()) -> let's remove this for now until we get to a patch/post request where we need it to keep our commits relevant to the code implemented